### ISSUE TITLE
training accuracy nearly zero

### ISSUE THREAD

**Issue by RunningLeon** [davidsandberg/facenet#745]
Hi,
First of all, many thanks for your support.
I tried to train a model using src\train_softmax.py, but the printed training accuracy was nearly zero. 
Thanks again.

Here's some detailed info:
1. Printed info during trainning
Epoch: [4][93/100]	Time 1.528	Loss 9.244	Xent 8.728	RegLoss 0.517	Accuracy 0.000	Lr 0.10000	Cl 0.853
Epoch: [4][94/100]	Time 1.448	Loss 9.142	Xent 8.626	RegLoss 0.517	Accuracy 0.000	Lr 0.10000	Cl 0.841
Epoch: [4][95/100]	Time 1.381	Loss 9.254	Xent 8.737	RegLoss 0.517	Accuracy 0.000	Lr 0.10000	Cl 0.857
Epoch: [4][96/100]	Time 1.397	Loss 9.092	Xent 8.575	RegLoss 0.516	Accuracy 0.030	Lr 0.10000	Cl 0.821
Epoch: [4][97/100]	Time 1.480	Loss 9.360	Xent 8.843	RegLoss 0.516	Accuracy 0.010	Lr 0.10000	Cl 0.883
Epoch: [4][98/100]	Time 1.567	Loss 9.211	Xent 8.695	RegLoss 0.516	Accuracy 0.000	Lr 0.10000	Cl 0.845
Epoch: [4][99/100]	Time 1.456	Loss 9.348	Xent 8.831	RegLoss 0.516	Accuracy 0.000	Lr 0.10000	Cl 0.875
Epoch: [4][100/100]	Time 1.553	Loss 9.182	Xent 8.666	RegLoss 0.516	Accuracy 0.020	Lr 0.10000	Cl 0.848
Saving variables
Variables saved in 0.04 seconds
Runnning forward pass on LFW images
........................
Accuracy: 0.57967+-0.03103
Validation rate: 0.00167+-0.00224 @ FAR=0.00100
....

2. arguments.txt
prelogits_hist_max: 10.0
prelogits_norm_p: 1.0
lfw_distance_metric: 1
model_def: models.squeezenet
moving_average_decay: 0.9999
lfw_use_flipped_images: True
validate_every_n_epochs: 1
log_histograms: False
lfw_subtract_mean: True
data_dir: ../data/webface-182x182
learning_rate_schedule_file: ../data/learning_rate_schedule.txt
learning_rate: 0.05
filter_filename: 
lfw_nrof_folds: 10
center_loss_factor: 0.0
center_loss_alfa: 0.95
learning_rate_decay_factor: 0.1
embedding_size: 128
max_nrof_epochs: 5
use_fixed_image_standardization: True
nrof_preprocess_threads: 4
models_base_dir: ../train/models
seed: 666
validation_set_split_ratio: 0.1
random_rotate: True
pretrained_model: None
min_nrof_val_images_per_class: 0
image_size: 160
learning_rate_decay_epochs: 5
filter_min_nrof_images_per_class: 0
lfw_batch_size: 10
weight_decay: 0.0004
logs_base_dir: ../train/reult/log
keep_probability: 1.0
gpu_memory_fraction: 0.85
random_crop: True
batch_size: 10
epoch_size: 10
prelogits_norm_loss_factor: 0.0
random_flip: True
filter_percentile: 100.0
lfw_pairs: ../data/pairs.txt
lfw_dir: ../data/lfw-182x182
optimizer: ADAGRAD

3, lfw_result.txt
0	0.56667	0.00933
10	0.57950	0.01033
20	0.58050	0.00767
30	0.57050	0.00500
40	0.57333	0.00600

**Comment by maryhh** [comment]
did you solve the problem? i  miss the same problem...

**Comment by Salary-only-17k** [comment]
Hi!!!
I have the problem.do you have a meth to do?

**Comment by hyeseongJeong** [comment]
I really want to know how to solve this problem
